_security/configuration/best-practices.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+title: Best Practices
+parent: Best Practices
```

Comment:
You can remove this, unless we plan on having additional best practice pages underneath this one. 

```suggestion

```